parsigs/parse_sig_api.py

```diff
@@ -61,6 +60,13 @@ class StructuredSig:
 inflect_engine = inflect.engine()
 
 
+def read_json_file():  # add parameters path, name
+    base_dir1 = os.path.dirname(__file__)
+    file_path1 = os.path.join(base_dir1, 'frequency_mapping.json')
+    with open(file_path1, 'r') as file1:
+        return json.load(file1)
+
+
 def _create_spell_checker():
     sc = SpellChecker()
     drug_words_frequency = Path(__file__).parent / 'resources/drug_names.txt'
```

Comment:
create `config.py` file - inside should be all program constant values for example fils paths that used across the program.